sort.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(readlink -e "$(dirname "${0}")")
+
+# Install jq
+if ! (command -v jq > /dev/null 2>&1); then
+    sudo apt install -y jq
```

Comment:
```suggestion
    sudo apt-get install -y jq
```
Minor: apt-get would be better for CLI environment.